tests/trainer/loops/test_evaluation_loop_flow.py

```diff
@@ -140,7 +140,7 @@ def backward(self, loss, optimizer, optimizer_idx):
     trainer.state.stage = RunningStage.TRAINING
     # make sure training outputs what is expected
     batch_idx, batch = 0, next(iter(model.train_dataloader()))
-    out = trainer.train_loop.run_training_batch(batch, batch_idx, 0)
+    out = trainer.fit_loop.training_loop.batch_loop.run(batch, batch_idx, 0)
```

Comment:
Should we have a property `trainer.train_loop = trainer.fit_loop.training_loop` and should we rename `training_loop` to `train_loop` for consistency ?

Reply:
@tchaton I don't think we should do this. There is not reason for users to actually engage with that loop and if we have a property on the trainer we open it to public API.

Reply:
I had such properties but also @ananthsub requested to remove them. 
But our plans was to expose the loops to the public so one way or another these interfaces will be accessible by users in the future.